src/IIIFPresentation/API/Features/Storage/Validators/FlatCollectionValidator.cs

```diff
@@ -9,18 +9,33 @@ public FlatCollectionValidator()
     {
         RuleSet("create", () =>
         {
-            RuleFor(a => a.Created).Empty().WithMessage("Created cannot be set");
-            RuleFor(a => a.Modified).Empty().WithMessage("Modified cannot be set");
-            RuleFor(a => a.Id).Empty().WithMessage("Id cannot be set");
-            RuleFor(a => a.Parent).NotEmpty().WithMessage("Creating a new collection requires a parent");
+            RuleFor(f => f.Created).Empty().WithMessage("Created cannot be set");
+            RuleFor(f => f.Modified).Empty().WithMessage("Modified cannot be set");
+            RuleFor(f => f.Id).Empty().WithMessage("Id cannot be set");
+            RuleFor(f => f.Parent).NotEmpty().WithMessage("Creating a new collection requires a parent");
         });
         
         RuleSet("update", () =>
         {
-            RuleFor(a => a.Created).Empty().WithMessage("Created cannot be set");
-            RuleFor(a => a.Modified).Empty().WithMessage("Modified cannot be set");
-            RuleFor(a => a.Id).Empty().WithMessage("Id cannot be set");
-            RuleFor(a => a.Parent).NotEmpty().WithMessage("Updating a collection requires a parent to be set");
+            RuleFor(f => f.Created).Empty().WithMessage("Created cannot be set");
```

Comment:
Would it be easier to use an alternative view-model/poco where these properties don't exist? Would mean that there's no way that they can be provided, which would reduce validation overhead? When viewing on swagger etc consumers would only see properties that can be updated, rather than a bunch that cannot. Would also avoid needing to keep this list updated if new properties were added.

Reply:
not sure if this needs another look, but I've updated this to use a POCO for `update` - I'm working on some fixes to `create` on another PR and will look to get this updated for `POST` along with those changes